### DISCUSSION TITLE
Display dags that are using a specififc connection

### DISCUSSION THREAD

**Discussion by shivannakarthik** [apache/airflow#36886] [Ideas]
On a large airflow instance, it is often important to understand which dags are using a specific connection so that rotation of the passwords/keys in the connection becomes easier and helps assess the impact.
Seeking feedback for this feature.

**Comment by potiuk** [comment]
There is no such feature. And it's not an easy one to get because connections are dynamically retrieved while tasks are running - especially when you use taskflow and PythonOperator.  Probably it would require to gather and update such information during DAG execution.
